lib/plugins/aws/deploy/lib/extendedValidate.test.js

```diff
@@ -113,6 +113,25 @@ describe('extendedValidate', () => {
       });
     });
 
+    it('should use function package level generated artifact if functions is packaged individiually', () => {
+      awsDeploy.serverless.service.package.individually = false;
+      stateFileMock.service.functions = {
+        first: {
+          package: {
+            individually: true,
+          },
+        },
+      };
+      fileExistsSyncStub.returns(true);
+      readFileSyncStub.returns(stateFileMock);
+
+      return awsDeploy.extendedValidate().then(() => {
+        expect(fileExistsSyncStub.calledTwice).to.equal(true);
+        expect(readFileSyncStub.calledOnce).to.equal(true);
+        expect(fileExistsSyncStub.lastCall.args[0]).to.have.string('.serverless/first.zip');
```

Comment:
@medikoo I'm not super happy about that assertion so please let me know if there are better ways to check it

Reply:
Probably better would be to just confirm that `awsDeploy.extendedValidate` doesn't crash with `individually` set on function level (which I believe will happen on `master` now)

Reply:
Done!